submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionRequestMongoPersistenceService.kt

```diff
@@ -67,32 +60,25 @@ class SubmissionRequestMongoPersistenceService(
         requestRepository.updateSubmissionRequestFile(requestFile.accNo, requestFile.version, requestFile.path, file)
     }
 
-    override suspend fun getPendingRequest(accNo: String, version: Int, handlerName: String): SubmissionRqt {
-        return getRequest(accNo, version, REQUESTED, handlerName)
-    }
-
-    override suspend fun getIndexedRequest(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, INDEXED, processId)
-    }
-
-    override suspend fun getLoadedRequest(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, LOADED, processId)
-    }
-
-    override suspend fun getCleanedRequest(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, CLEANED, processId)
-    }
-
-    override suspend fun getCheckReleased(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, CHECK_RELEASED, processId)
-    }
-
-    override suspend fun getFilesCopiedRequest(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, FILES_COPIED, processId)
-    }
-
-    override suspend fun getPersistedRequest(accNo: String, version: Int, processId: String): SubmissionRqt {
-        return getRequest(accNo, version, PERSISTED, processId)
+    override suspend fun getRqt(
```

Comment:
I'd call it `getSubmissionRequest` just to make clear which type of request it's